haystack/nodes/connector/crawler.py

```diff
@@ -236,18 +258,32 @@ def _write_to_files(
             else:
                 text = el.text
 
-            link_split_values = link.replace("https://", "").split("/")
-            file_name = f"{'_'.join(link_split_values)}.json"
-            file_path = output_dir / file_name
+            DocumentDict = Dict[str, Any]
 
-            data = {}
+            data: DocumentDict = {}
```

Comment:
This is really interesting. I didn't realize you could define types like this! :+1: 

If `DocumentDict` was reused it would be really a nice idea actually, but it seems to be used only once. Am I correct? If it's used only here, I think it's a bit overkill, because the type it replaces is not much longer than `DocumentDict` itself. So we should remove it. 

However I will keep it in mind to use it in other occasions when typing gets really long and repeats. There definitely are such cases in the codebase :smile: 

```suggestion
            data: Dict[str, Any] = {}
```

Reply:
Oh, yeah! I was testing the Crawler code using _mypy_, then I noticed there was this issue about `data` type. I did the long way :smile: